Library/Homebrew/cask/lib/hbc/cli/upgrade.rb

```diff
@@ -26,6 +26,7 @@ def run
           return
         end
 
+        ohai "Casks with `auto_updates` or `version :latest` will not be upgraded" if args.empty?
```

Comment:
Changed this from `auto_updates true` to just `auto_updates` to match what is displayed by `info`.